src/components/TopBar.js

```diff
@@ -27,6 +31,7 @@ function TopBar({ classes, menuAction }) {
         </IconButton>
 
         <Typography className={classes.title} variant="h6">Serif.nu</Typography>
+        <Typography className={classes.term} variant="h6" justify="right">Term: Spring 2019</Typography>
```

Comment:
Let's not do `variant="h6"` for this label. We want the hierarchy to emphasize the title on the left. Let's just go with the default typography variant (no variant prop needed).